lib/BrizyForms/Service/WebHooksService.php

```diff
@@ -0,0 +1,196 @@
+use BrizyForms\Model\Response;
+use BrizyForms\ServiceConstant;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+class WebHooksService extends Service
+{
+    /**
+     * @var Client
+     */
+    private $client;
+    /**
+     * @param FieldMap $fieldMap
+     * @param string $group_id
+     *
+     * @return mixed
+     */
+    protected function mapFields(FieldMap $fieldMap, $group_id = null)
+    {
+        foreach ($fieldMap->toArray() as $fieldLink) {
+            if ($fieldLink->getTarget() == ServiceConstant::AUTO_GENERATE_FIELD) {
+                $fieldLink->setTarget($fieldLink->getSourceTitle());
+            }
+        }
+        return $fieldMap;
+    }
+
+    /**
+     * @param FieldMap $fieldMap
+     * @param null $group_id
+     * @param array $data
+     * @param bool $confirmation_email$data
+     * @return mixed|void
+     * @throws ServiceException
+     * @throws \BrizyForms\Exception\FieldMapException
+     */
+    protected function internalCreateMember(FieldMap $fieldMap, $group_id = null, array $data = [], $confirmation_email = false)
+    {
+        $formFields = $fieldMap->transform($data, false);
+        $email = [];
+        if ($formFields->getEmail()) {
+            $email = ['Email' => $formFields->getEmail()];
+        }
+        $auth_data = $this->authenticationData->getData();
+        try {
+            $this->client->request($auth_data['request_method'], $auth_data['webhook_url'], [
+                'form_params' => $email
+            ]);
+        } catch (\Exception $e) {
+            throw new RequestException('Invalid request');
+        }
+    }
+
+    /**
+     * @param Folder|null $folder
+     * @return mixed|null
+     */
+    protected function internalGetGroups(Folder $folder = null)
+    {
+        return null;
+    }
+
+    /**
+     * @param Group $group
+     *
+     * @return mixed
+     */
+    protected function internalGetFields(Group $group = null)
+    {
+        return null;
+    }
+
+    /**
+     * @return bool
+     */
+    protected function hasValidAuthenticationData()
+    {
+        $data = $this->authenticationData->getData();
+        if (!isset($data['webhook_url']) || !isset($data['request_method'])) {
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * @return void
+     */
+    protected function initializeNativeService()
+    {
+    }
+
+    /**
+     * @param array $options
+     * @return RedirectResponse|Response|null
+     */
+    public function authenticate(array $options = null)
+    {
+        $data = $this->authenticationData->getData();
+
+        $this->client = new Client();
+
+        try {
+            $response = $this->client->request('GET', $data['webhook_url'], [
+                    'http_errors' => false,
+                    'timeout' => 3,
+                    'connect_timeout' => 1
+                ]);
+
+            if($response->getStatusCode() === 200)
```

Comment:
if you received a response == can connect to the host == success
no additional checks